Sources/StreamChat/Controllers/CurrentUserController/CurrentUserController.swift

```diff
@@ -252,6 +252,25 @@ public extension CurrentChatUserController {
             }
         }
     }
+    
+    /// Get all blocked users.
+    ///
+    /// - Parameter completion: Called when the API call is finished. Called with `Error` if the remote update fails.
+    ///
+    func getBlockedUsers(completion: @escaping (Result<[BlockedUser], Error>) -> Void) {
```

Comment:
In other places we use `loadXYZ`, what if we use the same term here as well for consistency?

Reply:
I think `get` here kinda makes sense, because we are just getting the blocked users, and not setting it anywhere 🤔 When we use `load` we usually assign the data into a property of the controller. But maybe for consistency, it is better we just name `load` 🤔

Reply:
Updated to `load` 👍